File: 백준/Silver/11399. ATM/ATM.py
import sys
input = sys.stdin.readline

M = int(input())
lst = list(map(int, input().split()))

lst.sort()

prev_sum = 0
res = 0

for x in lst:
    res = res + prev_sum + x
    prev_sum = prev_sum + x
    
print(res)

File: 백준/Silver/11399. ATM/ATM.java
     import java.util.*;

    public class Main {
        public static void main(String[] args) {
            Scanner sc = new Scanner(System.in);

            String[] input = sc.nextLine().split(" ");
            int N = Integer.parseInt(input[0]);

            input = sc.nextLine().split(" ");
            List<Integer> list = new ArrayList<>();

            for (String s: input) {
                list.add(Integer.parseInt(s));
            }

            Collections.sort(list);

            int sum = 0;
            int answer = 0;

            for (int number: list) {
                answer += number + sum;
                sum += number;
            }

            System.out.println(answer);
        }

    }
